src/Amadeus/Client/Struct/Fare/BaseMasterPricerMessage.php

```diff
@@ -96,12 +96,13 @@ protected function loadPassenger($passenger, &$counter, &$infantCounter)
      * @param FareMasterPricerTbSearch|FareMasterPricerCalendarOptions|TicketCheckEligibilityOptions $options
      * @return void
      */
-    protected function loadNrOfPaxAndResults($options)
+    protected function loadNrOfPaxAndResultsAndMultiTicketWeights($options)
```

Comment:
Oh god, these names are getting out of hand! :)

Reply:
I can change to `loadNumberOfUnits` 😄

Reply:
Nah, it's fine! Just when another option is added, we may have to reconsider the naming approach